dracut/30ignition/mnt-boot.mount

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Ignition Boot Mount
+ConditionPathExists=/dev/disk/by-label/boot
+
+# Wait for multipathd to start to prevent racing over
+# access to the raw device. This ensures that device symlinks
+# are setup properly.
+Wants=multipathd.service
+After=multipathd.service
+
+# Make sure that the device exists
+Requires=dev-disk-by\x2dlabel-boot.device
+After=dev-disk-by\x2dlabel-boot.device
```

Comment:
We also have `ConditionPathExists=/dev/disk/by-label/boot`. Feels like we shouldn't have both as then it is confusing if one of them is or isn't working for whatever reason.

Reply:
I was dealing with a race condition caused by multipath, however, since we're descoping that fo this change, dropped.